learn/tutorials-and-use-cases/_Quadratic-Linearly-Constrained-Binary-Optimization.ipynb

```diff
@@ -0,0 +1,372 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quadratic Linearly Constrained Binary Optimization\n",
+    "## Introduction\n",
+    "Quadratic Unconstrained Binary Optimization (Qubo) problems can be formulated from a square, symmetric objective function and a matrix of binary constraints. Suppose we are given an objective function, $O$, of dimension $n \\times n$,  and a set of $m$ constraints, represented by the matrix $A$, with dimension $m \\times n$ and right-hand side vector $b$ of length $m$. We want to combine them into a Qubo, which can be defined as $Q = O + \\alpha (A^T A-2\\mathrm{diag}(b^TA))$, where $\\alpha \\in \\mathbb{R}$. At this point, we can find an optimal solution,\n",
+    "$x^{*} = \\min_{x} x^T Q x$. \n",
+    "The parameter $\\alpha$ plays an important role in guaranteeing that the constraints are satisfied. We will not go into more detail on this page. \n",
+    "We will define a simple problem on the Upload tab and show how to upload the components.\n",
+    "Suppose the original problem we want to minimize is \n",
+    "$-3xy + xz,$\n",
+    "subject to the constraints $x + z = 1$ and  $2x + 2y= 2$.\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Uploading\n",
+    "There are three matrix components that can be associated with to a constraint problem of this type. The objective function in matrix form, the linear constraints matrix, and the right-hand side (RHS) represent the linear constraints themselves. The format should follow the transformation from $\\mathbf{A}\\vec{x}$ = $\\vec{b}$  &rarr;  $\\mathbf{A}\\vec{x}$ - $\\vec{b}$ = 0.\n"
```

Comment:
Remove the word "to" in "...can be associated with to a constraint problem of this type.”